lib/wanda/executions/evaluation.ex

```diff
@@ -173,6 +186,18 @@ defmodule Wanda.Executions.Evaluation do
     end
   end
 
+  defp maybe_add_failure_message(
+         %ExpectationEvaluation{type: :expect, return_value: false} = expectation_evaluation,
+         failure_message,
+         evaluation_scope
+       ) do
+    {:ok, interpolated_failure_message} = Rhai.eval("`#{failure_message}`", evaluation_scope)
```

Comment:
What about using the default message if an error is produced by `Rhai.eval` ?

Reply:
What's the default message? `Expectation not met`? We could go with that or any other hardcoded message like `Error interpolating message` or anything.
I just want to avoid some critical error that cancels the whole execution because somebody put `facts.timeuot` instead o` facts.timeout` hehe

Reply:
Nono you're actually right, I'd put `Expectation not met`